x-pack/solutions/security/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/customization_enabled/upgrade_notifications/configs/serverless.config.ts

```diff
@@ -13,4 +13,5 @@ export default createCompleteTierTestConfig({
     reportName:
       'Rules Management - Prebuilt Rules Upgrade Notifications (Customization Enabled) - Serverless Env',
   },
+  indexRefreshInterval: '1s',
```

Comment:
Was it necessary set 1 second index refresh interval for prebuilt rules upgrade notifications FTR tests because only these tests fail with the reduced value? We expect the these tests to work with the reduced refresh interval as well.

Reply:
I'll give another attempt with the reduced 50ms default